athenet/sparsifying/derest/activation.py

```diff
@@ -0,0 +1,27 @@
+"""For any neuron, we calculate its range of activation for inputs of neural
```

Comment:
we calculate (...)
It is not consistent with others. I think we should not write "we do sth" but "does sth" (file / module does) in every docstring